packages/backend/server/routes/publish.post.ts

```diff
@@ -10,16 +10,17 @@ import { generateTemplateHtml } from "~/utils/template";
 
 export default eventHandler(async (event) => {
   const origin = getRequestURL(event).origin;
-
   const {
     "sb-run-id": runIdHeader,
     "sb-key": key,
     "sb-shasums": shasumsHeader,
     "sb-comment": commentHeader,
     "sb-compact": compactHeader,
+    "sb-package-manager": packageManagerHeader,
   } = getHeaders(event);
   const compact = compactHeader === "true";
   const comment: Comment = (commentHeader ?? "update") as Comment;
+  const packageManager = packageManagerHeader || "npm";
```

Comment:
let's have a package manager of type PackageManager with possible values (npm, pnpm and yarn)